### PR TITLE
[TRAFODION-3178] Correct Syntax Descriptions of *CREATE PROCEDURE Statement* and Fix Typos in *Trafodion SQL Reference Manual*

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_clauses.adoc [hunk 1]
```diff
@@ -1053,8 +1053,7 @@ expressions in each item must be the same as the number of value columns
 in the column list.
 +
 In the example TRANSPOSE A,B,C AS V, the items are A,B, and C, and the
-value column is V. This form can be thought of as a shorter way of writing TRANSPOSE
-(A),(B),(C) AS (V).
+value column is V. This form can be thought of as a shorter way of writing TRANSPOSE (A),(B),\(C) AS (V).
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_clauses.adoc]: Not sure why the slash is before "(C)"; I don't think it is correct. Or is the slash an escape? Maybe "(C)" is interpreted as the copyright symbol by the rendering software?

File: docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc [hunk 2]
```diff
@@ -31,20 +31,27 @@ and expressions that you can use in {project-name} SQL statements. The
...
+[[categories]]
+== Categories
 
-For datetime functions, the definition of standard normalization is: If
-the ending day of the resulting date is invalid, the day will be rounded
-DOWN to the last day of the result month.
+Use these types of functions within an SQL value expression:
 
-== Aggregate (Set) Functions
+** Aggregate (Set) Functions
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc]: Do you want to include LOB Functions and Encryption Functions in this list?

File: docs/sql_reference/src/asciidoc/_chapters/sql_clauses.adoc (hunk 1 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_clauses.adoc]: Yes, the slash is an escape, (C) is rendered as © 😁 

c

escape

File: docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc (hunk 2 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc]: @DaveBirdsall Thanks, I've added 😁

**Comment by liuyu000** [comment]
@DaveBirdsall Thanks, I'll merge and publish web updates 😁

**Pull request merged**